src/Microsoft.DocAsCode.Build.ReferenceBase/ReferenceDocumentProcessorBase.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.ReferenceBase
+{
+    using System;
+    using System.Collections.Immutable;
+
+    using Microsoft.DocAsCode.Build.Common;
+    using Microsoft.DocAsCode.Plugins;
+
+    /// <summary>
+    /// Base document processor for reference.
+    /// </summary>
+    public abstract class ReferenceDocumentProcessorBase : DisposableDocumentProcessor
```

Comment:
>ReferenceDocumentProcessorBase [](start = 26, length = 30)

looks to me the base class simply separate build steps into more build steps, instead of sharing common behaviors, do we really need this base class? or DisposableDocumentProcessor is enough?